app/assets/javascripts/pdfjs_viewer/viewer.js.erb

```diff
@@ -7141,17 +7141,19 @@ var PDFViewerApplication = {
 };
 window.PDFView = PDFViewerApplication; // obsolete name, using it as an alias
 
+var HOSTED_VIEWER_ORIGINS = <%= ENV.fetch("PDFJS_VIEWER_ORIGINS", "").split(",").inspect %>;
 
-var HOSTED_VIEWER_ORIGINS = ['null',
-  'http://mozilla.github.io', 'https://mozilla.github.io'];
 function validateFileURL(file) {
   try {
+    // IE workaround because window.location.origin
+    // is not available in < IE 11
```

Comment:
No need for a comment. We don't want to own this code. We just use what we get from PDF.js